src/middleware/api.js

```diff
@@ -8,7 +8,13 @@ import jwtDecode from 'jwt-decode';
 
 import { settings } from '~/config';
 
-import { LOGIN } from '@plone/volto/constants/ActionTypes';
+import {
+  LOGIN,
+  RESET_APIERROR,
+  SET_APIERROR,
+} from '@plone/volto/constants/ActionTypes';
+
+const ACTIONS_RAISING_ERRORS = ['GET_CONTENT', 'UPDATE_CONTENT'];
```

Comment:
Could this be read from config.settings? This way it would be extensible without customizing api.js